etl-la-porte-bleue/serverless.yml

```diff
@@ -0,0 +1,22 @@
+org: mialesta
+app: etl-la-porte-bleue
+service: etl-la-porte-bleue
+
+frameworkVersion: '3'
+
+provider:
+  name: aws
+  runtime: python3.8
+  environment:
+    MONGO_DB: ${file(./serverless.env.yml):MONGO_DB, ''}
+    POSTGRES_DB: ${file(./serverless.env.yml):POSTGRES_DB, ''}
```

Comment:
are you sure about that syntax?
from their docs, I see this https://www.serverless.com/framework/docs/providers/aws/guide/variables#referencing-environment-variables
```suggestion
    MONGO_DB: ${env:MONGO_DB, ''}
    POSTGRES_DB: ${env:POSTGRES_DB, ''}
```
for env files 
and https://www.serverless.com/framework/docs/providers/aws/guide/variables#reference-properties-in-other-files
for yaml files.
are you sure your yaml file is correct? 
```yaml
# serverless.env.yml
MONGO_DB: xxxx
POSTGRES_DB: yyyy
```
note the `:`,  not a `=` like in env files